frontend/src/components/SizeFilter.cy.js

```diff
@@ -0,0 +1,20 @@
+/* global cy */
+
+import React from 'react'
+import SizeFilter from './SizeFilter'
+import '../index.css'
+
+const sizes = ['All', 'Small', 'Medium', 'Enterprise', 'Large Enterprise', 'Very Large Enterprise']
+
+describe('SizeFilter component', () => {
+  it("Filters by each size ('All', 'Small', 'Medium', 'Enterprise', 'Large Enterprise', and 'Very Large Enterprise')", () => {
+    sizes.forEach((size) => {
+      const onChangeStub = cy.stub().as('onChangeStub')
+
+      cy.mount(<SizeFilter sizeFilter="All" onChange={onChangeStub} />)
+
+      cy.get('[data-testid="filter"]').select(size)
```

Comment:
Eu deixaria um espaço em branco entre _Act_ e _Assert_.

```suggestion
      cy.get('[data-testid="filter"]').select(size)
      
```